pkg/prowgen/prowgen_config.go

```diff
@@ -53,6 +53,7 @@ type IgnoreConfigs struct {
 
 type Promotion struct {
 	Namespace string `json:"namespace,omitempty" yaml:"namespace,omitempty"`
+	SkipSrc   bool   `json:"skipSrc,omitempty" yaml:"skipSrc,omitempty"`
```

Comment:
Added this `bool` to optionally skip src image promotion. We don't utilize it in client.